install/prepare_site_init.md

```diff
@@ -452,7 +453,28 @@ the system size.
         "$min_amount"
     done
     ```
+### Enable Rack Resiliency
 
+This section can be skipped if Rack Resiliency feature is not required.
+
+1. (`pit#`) Update the customizations file.
+
+    ```bash
+      yq write -i ${SITE_INIT}/customizations.yaml \
+        'spec.kubernetes.services.rack-resiliency.enabled' "true"
+    ```
+1. (`pit#`) If site specific prefix for Kubernetes and `Ceph` zones is required:
```

Comment:
This also needs to provide some way for an admin to make this decision. How do they know if it is required? Again, this is covered by the "minimal PR" I outlined for you in Slack.